config/packages/prod/deprecations.yaml

```diff
@@ -0,0 +1,8 @@
+# As of Symfony 5.1, deprecations are logged in the dedicated "deprecation" channel when it exists
+#monolog:
+#    channels: [deprecation]
+#    handlers:
+#        deprecation:
+#            type: stream
+#            channels: [deprecation]
+#            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
```

Comment:
Why is this commented out? Isn't this replacing the deprecated stuff removed from the config/packages/<mode>/monolog.yaml?

Reply:
This is all on by default

Reply:
It may not be on by default, but I think it's ok for now. We can decide if we even need this.